examples/grid_examples/PyFstat_example_grid_search_F0.py

```diff
@@ -69,11 +69,10 @@
 )
 search.run()
 
+print("Plotting 2F(F0)...")
 fig, ax = plt.subplots()
-xidx = search.keys.index("F0")
-frequencies = np.unique(search.data[:, xidx])
-twoF = search.data[:, -1]
-
+frequencies = search.data["F0"]
+twoF = search.data["twoF"]
```

Comment:
this is the basic example for why the change to the internal data structures makes usage easier